### PR TITLE
11169 fix new btn

### PR DESCRIPTION
**Pull request by petrican**
https://www.loom.com/share/eb1ebc2e5ef44e3aa669185aafac390a

### CODE REVIEW

File: frontend/src/components/widget/InlineTabWrapper.js [hunk 1]
```diff
@@ -114,6 +116,16 @@ class InlineTabWrapper extends PureComponent {
...
+            // calling getData to have the updated included tabs
+            getData({
+              entity: 'window',
+              docType: windowId,
+              docId: docId,
+            }).then(({ data }) => {
+              const response = data[0];
+              response.includedTabsInfo &&
+                updateDataIncludedTabsInfo('master', response.includedTabsInfo);
+            });
```
Comment (teosarca) [inline frontend/src/components/widget/InlineTabWrapper.js:128]: IMHO another API call is not needed. You already have the info returned by DELETE endpoint.

image

image

**Review by teosarca — CHANGES_REQUESTED** [review]
check my comments

File: frontend/src/components/widget/InlineTabWrapper.js (hunk 1 above)
Comment (petrican) [reply frontend/src/components/widget/InlineTabWrapper.js]: You are right. Just pushed fix . Pls re-check

**Review by siemiatj — APPROVED** [review]
Finally !! 👯

**Comment by metas-Kay** [comment]
related to https://github.com/metasfresh/metasfresh/issues/11169

**Pull request merged**